### DISCUSSION TITLE
how to create mailbox and set protocol imap

### DISCUSSION THREAD

**Discussion by githubsharezone** [postalserver/postal#2407] [Help with using Postal]
with postal is possible create a mailbox and set the imap protocol

**Comment by willpower232** [comment]
This is not possible, Postal has no concept of mailboxes and is not designed to keep copies of messages long term. Postal allows things to send messages and if it receives a reply, can direct that reply to some code designed to handle it.

**Comment by githubsharezone** [comment]
thank you very much for the clarification
